src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -412,6 +412,7 @@
     <OutOfBoxFramework Include="netcore50">
       <RuntimeId>win10</RuntimeId>
     </OutOfBoxFramework>
+    <OutOfBoxFramework Condition="'$(IsUap101Defined)'=='true'" Include="uap10.1"/>
```

Comment:
It's going to be defined in packaging.props on corefx repo on relevant branches where uap10.1 is 'supported' (i.e.: dev/api). 
for ref: https://github.com/SedarG/corefx/commit/cf8d4c0e4a1c5758e27cfa41a08a45e85f491df9